src/core/dbus-cgroup.c

```diff
@@ -826,6 +829,31 @@ int bus_cgroup_set_property(
 
                 return 1;
 
+        } else if (STR_IN_SET(name, "MemoryLow", "MemoryHigh", "MemoryMax")) {
```

Comment:
* I'm kinda confused about "\n" in unit_write_drop_in_private_format(). There seem to be other usages which don't have it either. Will play with them.
* MemoryMax and MemoryLimit are equivalent and translated back and forth. The legacy path uses MemoryMax if MemoryLimit is not set.
* As for warning messages, I'm not sure what to do as it can add up to a lot of noise about various situations (translated, not translated, translated with range truncated, switching from translating to not doing it). I'm thinking about adding a one-time warning which warns about using mismatching version of configs and that only basic translations are available.